portfolio/src/main/webapp/index.html

```diff
@@ -5,21 +5,35 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+    body {
+      background-image: url('https://i.pinimg.com/originals/92/17/4a/92174ad3758fdb8f9c2e4579ad5430a0.jpg');
+      background-size: cover;
+    }
+    </style> -->
   </head>
   <body>
+    <div class="header" style="background-image: url('https://wallpaperaccess.com/full/1425559.jpg'); background-size: cover;">
+    <a href="#default" class="logo"><img src=/images/logo.png alt="My logo" width="70" height="70">
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#General_Formatting:

Use a new line for every block, list, or table element, and indent every such child element.